lb.tf

```
@@ -0,0 +1,11 @@
+module "lb" {
+  source = "git::https://github.com/appuio/terraform-modules.git//modules/lb-cloudscale?ref=skeleton-and-lb"
```

Probably makes sense to make the module version configurable

```suggestion
  source = "git::https://github.com/appuio/terraform-modules.git//modules/lb-cloudscale?ref=${var.module_versions.lb_cloudscale}"
```

---

Sadly:

```
│ Error: Unsuitable value type
│ 
│   on lb.tf line 2, in module "lb":
│    2:   source = "git::https://github.com/appuio/terraform-modules.git//modules/lb-cloudscale?ref=${var.module_versions.lb_cloudscale}"
│ 
│ Unsuitable value: value must be known
```

Makes sense since `terraform init` takes no variables

---

You need to define the variable in `variables.tf`, then it should work just fine, something like

```
variable module_versions {
  type    = map(object({ lb_cloudscale = string }))
  default = {
    lb_cloudscale = skeleton-and-lb
  }
}
```

---

That's exactly what i did. Won't work. `terraform init` is separate from `plan` and `apply`

---

Oof, I see. I guess we'll have to live with that then. However in that case, I'd strongly prefer to have a "real" version for the terraform-modules.git repo before merging this PR.